src/main/java/uk/gov/register/presentation/Proofs.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.register.presentation;
+
+import uk.gov.register.proofs.ct.SignedTreeHead;
+
+public class Proofs {
+    private SignedTreeHead certificateTransparencySignedTreeHead;
+
+    public SignedTreeHead getCertificateTransparencySignedTreeHead() {
+        return certificateTransparencySignedTreeHead;
+    }
+
+    public void setCertificateTransparencySignedTreeHead(SignedTreeHead certificateTransparencySignedTreeHead) {
```

Comment:
our jackson data objects so far have used constructor injection rather than setter injection. could this class do the same for consistency?